openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -489,92 +490,92 @@
+    "create-or-update-email-account-for-bot": "Changing the account email will update or create a new bot user.",
     "created-this-task-lowercase": "created this task",
-    "data-asset-has-been-action-type": "Data asset has been {{actionType}}",
+    "data-asset-has-been-action-type": "Data Asset has been {{actionType}}",
     "data-insight-page-views": "Displays the number of times a dataset type was viewed.",
     "data-insight-subtitle": "Get a single pane view of the health of all your data assets over time.",
     "dbt-run-result-http-path-message": "dbt run results http path to extract the test results information.",
     "deeply-understand-table-relations-message": "Deeply understand table relations; thanks to column-level lineage.",
     "delete-action-description": "Deleting this {{entityType}} will permanently remove its metadata from OpenMetadata.",
-    "delete-entity-permanently": "Once you delete this {{entityType}}, it will be removed permanently",
+    "delete-entity-permanently": "Once you delete this {{entityType}}, it will be removed permanently.",
     "delete-message-question-mark": "Delete Message?",
     "delete-team-message": "Any teams under \"{{teamName}}\" will be {{deleteType}} deleted as well.",
     "delete-webhook-permanently": "You want to delete webhook {{webhookName}} permanently? This action cannot be reverted.",
-    "discover-your-data-and-unlock-the-value-of-data-assets": "Discover your data and unlock the value of data assets",
-    "email-is-invalid": "Email is invalid.",
+    "discover-your-data-and-unlock-the-value-of-data-assets": "Discover your data and unlock the value of data assets.",
+    "email-is-invalid": "Invalid Email.",
     "enable-column-profile": "Enable column profile",
-    "enables-end-to-end-metadata-management": "Enables end-to-end metadata management with data discovery, data duality, observability, and people collaboration",
-    "enter-column-description": "Enter Column Description",
+    "enables-end-to-end-metadata-management": "Enables end-to-end metadata management with data discovery, data quality, observability, and people collaboration",
+    "enter-column-description": "Enter column description",
     "enter-comma-separated-term": "Enter comma separated term",
     "enter-display-name": "Enter display name",
     "enter-feature-description": "Enter feature description",
     "enter-interval": "Enter interval",
     "enter-test-case-name": "Enter test case name",
     "entity-already-exists": "{{entity}} already exists.",
-    "entity-owned-by-name": "This Entity is Owned by {{entityOwner}}",
+    "entity-owned-by-name": "This entity is owned by {{entityOwner}}",
     "entity-restored-error": "Error while restoring {{entity}}",
     "entity-restored-success": "{{entity}} restored successfully",
     "fetch-dbt-files": "Available sources to fetch dbt catalog and manifest files.",
```

Comment:
Would it be correct to say:

These are the available sources to fetch dbt catalog and manifest files.